flink-runtime/src/main/java/org/apache/flink/runtime/rest/handler/job/savepoints/SavepointHandlers.java

```diff
@@ -109,9 +97,7 @@
  * }
  * </pre>
```

Comment:
You removed the trigger id from the savepoint response but the Javadoc still has it:
```
 * <pre>
 * {
 *     "status": {
 *         "id": "COMPLETED"
 *     },
 *     "savepoint": {
 *         "request-id": "7d273f5a62eb4730b9dea8e833733c1e",
 *         "location": "/tmp/savepoint-d9813b-8a68e674325b"
 *     }
 * }
```

Reply:
Will remove it.